pkg/solana/chainreader/client_wrapper.go

```diff
@@ -0,0 +1,42 @@
+package chainreader
+
+import (
+	"context"
+
+	"github.com/gagliardetto/solana-go"
+	"github.com/gagliardetto/solana-go/rpc"
+)
+
+type RPCClientWrapper struct {
+	*rpc.Client
+}
+
+func (w *RPCClientWrapper) GetMultipleAccountData(ctx context.Context, keys ...solana.PublicKey) ([][]byte, error) {
```

Comment:
I don't see this being used, is the idea to use it for batching?

Reply:
The `RPCClientWrapper` is to satisfy the contract reader service expected interface. I had to use an interface here instead of a client directly because there was something in the response object that isn't directly exported from the solana rpc client, making it difficult to write unit tests for.